PhysicsTools/PatAlgos/python/tools/jetTools.py

```diff
@@ -201,112 +201,141 @@ def setupBTagging(process, jetSource, pfCandidates, explicitJTA, pvSource, svSou
-                        extSVCollection=svSource
-                        )
-                    )
-                if svClustering or fatJets != cms.InputTag(''):
-                    setupSVClustering(getattr(process, btagInfo+labelName+postfix), svClustering, algo, rParam, fatJets, groomedFatJets)
+                    setupSVClustering(getattr(process, btagPrefix+btagInfo+labelName+postfix), svClustering, algo, rParam, fatJets, groomedFatJets)
             if btagInfo == 'pfInclusiveSecondaryVertexFinderNegativeCvsLTagInfos':
-                 setattr(
-                    process,
-                    btagInfo+labelName+postfix,
-                    btag.pfInclusiveSecondaryVertexFinderNegativeCvsLTagInfos.clone(
-                        trackIPTagInfos = cms.InputTag('pfImpactParameterTagInfos'+labelName+postfix),
-                        extSVCollection=svSource
-                        )
-                    )
-                 if svClustering or fatJets != cms.InputTag(''):
-                     setupSVClustering(getattr(process, btagInfo+labelName+postfix), svClustering, algo, rParam, fatJets, groomedFatJets)
+                setattr(process, btagPrefix+btagInfo+labelName+postfix, btag.pfInclusiveSecondaryVertexFinderNegativeCvsLTagInfos.clone(trackIPTagInfos = cms.InputTag(btagPrefix+'pfImpactParameterTagInfos'+labelName+postfix), extSVCollection=svSourceCvsL))
+                if svClustering or fatJets != cms.InputTag(''):
+                    setupSVClustering(getattr(process, btagPrefix+btagInfo+labelName+postfix), svClustering, algo, rParam, fatJets, groomedFatJets)
             if btagInfo == 'pfGhostTrackVertexTagInfos':
-                setattr(process, btagInfo+labelName+postfix, btag.pfGhostTrackVertexTagInfos.clone(trackIPTagInfos = cms.InputTag('pfImpactParameterTagInfos'+labelName+postfix)))
+                setattr(process, btagPrefix+btagInfo+labelName+postfix, btag.pfGhostTrackVertexTagInfos.clone(trackIPTagInfos = cms.InputTag(btagPrefix+'pfImpactParameterTagInfos'+labelName+postfix)))
             if btagInfo == 'pfGhostTrackTagVertexInfosAK8':
-                setattr(process, btagInfo+labelName+postfix, btag.pfGhostTrackVertexTagInfosAK8.clone(trackIPTagInfos = cms.InputTag('pfImpactParameterTagInfosAK8'+labelName+postfix)))
+                setattr(process, btagPrefix+btagInfo+labelName+postfix, btag.pfGhostTrackVertexTagInfosAK8.clone(trackIPTagInfos = cms.InputTag(btagPrefix+'pfImpactParameterTagInfosAK8'+labelName+postfix)))
             if btagInfo == 'pfSecondaryVertexNegativeTagInfos':
-                setattr(process, btagInfo+labelName+postfix, btag.pfSecondaryVertexNegativeTagInfos.clone(trackIPTagInfos = cms.InputTag('pfImpactParameterTagInfos'+labelName+postfix)))
+                setattr(process, btagPrefix+btagInfo+labelName+postfix, btag.pfSecondaryVertexNegativeTagInfos.clone(trackIPTagInfos = cms.InputTag(btagPrefix+'pfImpactParameterTagInfos'+labelName+postfix)))
```

Comment:
In principle, yes, it should be added here as well. Good catch. Will update as well.